src/client/components/app/ScanNetworkIcon/index.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+import { GeneralVaultView, Network } from '@types';
+import { DefaultscanIcon, EtherscanIcon, FtmscanIcon, Icon } from '@components/common';
+
+interface ScanNetworkIconProps {
+  currentNetwork?: Network;
+  blockExplorerUrl?: string;
+  selectedVault: GeneralVaultView;
+}
+
+const IconScan = styled(Icon)`
+  fill: black;
+  padding-left: 5px;
+  padding-bottom: 4px;
+  &:hover {
+    cursor: pointer;
+  }
+`;
+
+export const ScanNetworkIcon = ({ currentNetwork, blockExplorerUrl, selectedVault }: ScanNetworkIconProps) => {
+  const handleScanSiteExplorer = () => {
+    switch (currentNetwork) {
+      case 'mainnet':
+        return window.open(`${blockExplorerUrl}${selectedVault.address}`);
```

Comment:
couldn't we just remove the switch here and always do window.open? @TacosTonight